etna/models/base.py

```diff
@@ -820,4 +820,70 @@ def get_model(self) -> "DeepBaseNet":
         return self.net
 
 
-BaseModel = Union[PerSegmentModel, PerSegmentPredictionIntervalModel, MultiSegmentModel, DeepBaseModel]
+class MultiSegmentPredictionIntervalModel(FitAbstractModel, PredictIntervalAbstractModel, BaseMixin):
```

Comment:
It seems we need just empty class without any implementation.

Downstream classes has implemented own logic already. Seems we can't share it